src/stac_utils/ngpvan.py

```diff
@@ -151,3 +151,84 @@ def get_paginated_items(self, url, **kwargs):
+            "contactMode": "Person",
+        }
+
+        if has_identifier and id_key is not None:
+            formatted_json["identifiers"] = [
+                {"type": "votervanid", "externalId": row.get(id_key)}
+            ]
+        elif has_identifier and id_key is None:
+            raise ValueError("did not indicate name of id key column")
+        elif row.get("custom_field_id") and row.get("custom_field_group_id"):
+            formatted_json["customFieldValues"] = [
+                {
+                    "custom_field_id": row.get("custom_field_id"),
+                    "custom_field_group_id": row.get("custom_field_group_id"),
+                    "assignedValue": row.get(id_key),
+                }
+            ]
+        else:
+            print("No ID key used")
+
+        if row.get("email"):
+            formatted_json["emails"] = [{"email": row.get("email")}]
+
+        if row.get("phone"):
+            formatted_json["phones"] = [
+                {"phoneNumber": str(row.get("phone")).replace(".0", "")}
+            ]
+
+        if row.get("middle_name"):
+            formatted_json["middleName"] = row.get("middle_name")
+
+        # if row.get('street_address') or row.get('city') or row.get('stateOrProvince') or row.get('zipOrPostalCode'):
+        address = {}
+
+        if row.get("street_address"):
+            address["addressLine1"] = row.get("street_address")
+
+        if row.get("city"):
+            address["city"] = row.get("city")
+
+        if row.get("state") or row.get("stateOrProvince"):
+            if row.get("state"):
+                address["stateOrProvince"] = row.get("state")
+            else:
+                address["stateOrProvince"] = row.get("stateOrProvince")
+
+        if row.get("zip") or row.get("zipOrPostalCode"):
+            if row.get("zip"):
+                address["zipOrPostalCode"] = row.get("zip")
+            else:
+                address["zipOrPostalCode"] = row.get("zipOrPostalCode")
+
+        if address is not {}:
+            formatted_json["addresses"] = [address]
+
+        return strip_dict(formatted_json)
+
+    def van_phone_validation(phone) -> str:
+        """
+        This method validates phone numbers using VAN's API, and if number is not valid the phone variable
+        is assigned an empty string
+        :param phone: str, phone number from ActionKit
+        :return: str, empty string if phone number was not valid, or returns the valid phone number
+        """
+        van_phone_endpoint = "https://api.securevan.com/v4/people/findByPhone"
+        payload = {"phoneNumber": f"{phone}"}
+        headers = {
+            "Accept": "application/json",
+            "Content-Type": "application/json",
+            "Authorization": os.getenv("VAN_AUTH")
+        }
+        response = requests.post(van_phone_endpoint, json=payload, headers=headers)
+        if response.status_code == 400:
+            phone = ""
+        return phone
```

Comment:
this shouldn't need its own requests session, it should just be a convenience wrapper for self.post('/people/findByPhone', body={"phoneNumber": f"{phone}"})